docs/checks/internal/checks.yaml

```diff
@@ -78,6 +78,10 @@ checks:
         be enabled for forks where security updates have ever been turned on so projects
         maintaining stable forks should evaluate whether this behavior is satisfactory
         before turning it on.
+      - >-
+        Unlike dependabot, renovatebot has support to migrate dockerfiles' dependencies from version pinning to hash pinning 
```

Comment:
Nit: might come across as if we are putting down dependabot. Consider changing to:

```
Renovatebot supports migrating Dockerfile dependencies from ...
```

Reply:
I thought `conversation resolution before merging` should stop this PR from being merged, lol.

Reply:
it should have!